lib/CM5Receiver/CM5Receiver.circuit.tsx

```diff
@@ -0,0 +1,11 @@
+import { CM5Connector } from "./CM5Connector"
+import type { ChipProps } from "@tscircuit/props"
+
+export const CM5Receiver = (props: ChipProps) => {
+  return (
+    <group>
+      <CM5Connector side="right" name="JP1" />
+      <CM5Connector side="left" schX={-3.01} pcbX={-33.98} name="JP2" />
```

Comment:
the way you've defined this removes all the type safety, the connectors have to be defined separately or with a generic type in order to preserve type safety